### PR TITLE
USI not to use background shell

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: usi/src/main/java/daq/usi/ovs/OpenVSwitch.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 public class OpenVSwitch implements SwitchController {
 
-  private static final String OVS_OUTPUT_FILE = "ovs_output.txt";
+  private static final String OVS_OUTPUT_FILE = "sec.ofctl";
```
Comment (henry54809) [inline usi/src/main/java/daq/usi/ovs/OpenVSwitch.java:22]: Also can you please change the test filename? usi/src/test/resources/ovs_output.txt

Comment (grafnu) [reply usi/src/main/java/daq/usi/ovs/OpenVSwitch.java:22]: Do you know why the test didn't fail?  Something like that should have been caught, so seems like something's misconfigured.

Comment (henry54809) [reply usi/src/main/java/daq/usi/ovs/OpenVSwitch.java:22]: The unit tests aren't setup to run but they will be as part of the github actions migration.

Comment (grafnu) [reply usi/src/main/java/daq/usi/ovs/OpenVSwitch.java:22]: Ok, renamed test resource... PTAL.

**Pull request merged**